### DISCUSSION TITLE
Programas de TV

### DISCUSSION THREAD

**Discussion by thomraider12** [LITUATUI/M3UPT#91] [General]
Boas,
Os programas de televisão que estão na lista m3u, são inseridos com um actions, ou manualmente?
E se manualmente, onde é que os posso ir buscar?

Obrigado.

**Comment by LITUATUI** [comment]
Olá, são inseridos manualmente.

Podes obter os links facilmente com uma extensão [deste](https://chromewebstore.google.com/detail/m3u8-sniffer-tv-find-and/akkncdpkjlfanomlnpmmolafofpnpjgn?pli=1) tipo.

**Reply by thomraider12** [reply]
Obrigado.
